src/Model/Filter/Base.php

```diff
@@ -90,9 +90,19 @@ public function __construct($name, Manager $manager, array $config = [])
     /**
      * Get the manager.
      *
-     * @return \Search\Manager
+     * @deprecated Use getManager()
      */
     public function manager()
+    {
+        return $this->getManager();
+    }
+
+    /**
+     * Get the manager.
+     *
+     * @return \Search\Manager
+     */
+    public function getManager()
```

Comment:
There's no setter for the manager so adding "get" prefix to the getter method is unnecessary. Let's avoid unnecessary method name changes.

Reply:
Well, I disagree with that, also on the framework level. It's weird.